quinn-proto/src/token.rs

```diff
@@ -19,79 +19,79 @@ use crate::{
 // - AEAD nonce is always set to 0
 // in other words, for each ticket, use different key derived from random using HKDF
 
-pub fn generate<K>(
-    key: &K,
-    address: &SocketAddr,
-    src_cid: &ConnectionId,
-    dst_cid: &ConnectionId,
-    issued: SystemTime,
-) -> Vec<u8>
-where
-    K: HmacKey,
-{
-    let mut buf = Vec::new();
+pub struct Token {
```

Comment:
Let's call this `RetryToken` instead to more clearly distinguish it from `ResetToken`? Potentially move both into the same module renamed to `tokens`?